contrib/depends/packages/openssl.mk

```diff
@@ -5,8 +5,8 @@ $(package)_file_name=$(package)-$($(package)_version).tar.gz
 $(package)_sha256_hash=88525753f79d3bec27d2fa7c66aa0b92b3aa9498dafd93d7cfa4b3780cdae313
 
 define $(package)_set_vars
-$(package)_config_env=AR="$($(package)_ar)" ARFLAGS=$($(package)_arflags) RANLIB="$($(package)_ranlib)" CC="$($(package)_cc)"
-$(package)_config_env_android=ANDROID_NDK_ROOT="$(host_prefix)/native" PATH="$(host_prefix)/native/bin" CC=clang AR=ar RANLIB=ranlib
+$(package)_config_env=AR="$($(package)_ar)" RANLIB="$($(package)_ranlib)" CC="$($(package)_cc)"
+$(package)_config_env_android=ANDROID_NDK_ROOT="$(host_prefix)/native" PATH="$(host_prefix)/native/bin" CC=clang AR=ar RANLIB="ranlib -D"
```

Comment:
Thanks for spotting this issue and submitting the PR. In my experience, with subtle issues like this one, a comment here in the source (describing the rationale for the flag, which you could probably just lift from the PR) would help future readers save a lot of time (the fact that the documentation and behavior drifts across platforms makes it very hard for a new contributor to understand the issue).

Reply:
I added a comment.